src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -71,6 +71,8 @@ public static partial class PlatformDetection
                 ? dlopen("libgdiplus.dylib", RTLD_LAZY) != IntPtr.Zero
                 : dlopen("libgdiplus.so", RTLD_LAZY) != IntPtr.Zero || dlopen("libgdiplus.so.0", RTLD_LAZY) != IntPtr.Zero;
 
+        public static bool IsInContainer => RuntimeInformation.IsOSPlatform(OSPlatform.Linux) && File.Exists("/.dockerenv");
```

Comment:
Why is this within the #else statement of a netcoreapp30 #if ? Was this intentional?

Reply:
no, it was not intentional. It seems like it got broken by merge from upstream. It seems like we should move it above IsDrawingSupported...